rest/spring-rest-api/src/main/java/io/egen/api/util/UtilServices.java

```diff
@@ -0,0 +1,42 @@
+package io.egen.api.util;
+
+import java.util.List;
+
+import org.springframework.stereotype.Component;
+
+import io.egen.api.entity.WeatherDetails;
+import io.egen.api.entity.WindDetails;
+
+@Component
```

Comment:
There is no point of having it as a Component if only members that this class contains are static.